rise-google-sheet.html

```diff
@@ -76,7 +76,7 @@
 </dom-module>
 
 <!-- build:version -->
-<script>var sheetVersion = "2.3.0";</script>
+<script>var sheetVersion = "2.4.1";</script>
```

Comment:
@stulees Ah, might have to figure out a better solution for this at some point then.